src/aging_session/modify_user_session.clj

```diff
@@ -0,0 +1,11 @@
+(ns aging-session.modify-user-session)
+
+(defprotocol ModifyUserSession
+  "A protocol for modifying the session(s) of a user who is not currently logged in and
+   controlling the application."
+
+  (terminate-user-session
+    [store x]
+    [store x y]
```

Comment:
Just do (terminate-user-session session-key), perhaps, and pull out the logic for transforming user-id + firm-id -> session key into another protocol (could even do that in de.web TBH)

Reply:
Cool. I will be doing as discussed. Thanks!